75 launcher io layers

Something-Inconspicuous opened the pull request:
Merging into the branch it branched off of, then we'll merge into BoB-dev.

falOn-Dev committed:
Some AdvantageKit stuff

Also autos is broken. Too bad!

falOn-Dev committed:
It will be rewritten

trust

falOn-Dev committed:
Update Robot.kt

Something-Inconspicuous committed:
LauncherIO interface and inputs

Something-Inconspicuous committed:
package

Something-Inconspicuous committed:
Better IO layer to match Falon's conventions

Something-Inconspicuous committed:
roller pid and ff methods

Something-Inconspicuous committed:
literally 4 characters

unless you count spaces. then it's 6

Something-Inconspicuous committed:
First attempt at impl, yell at me when its wrong

Something-Inconspicuous committed:
Separation of flywheels

Something-Inconspicuous committed:
Finished Separating flywheels

Something-Inconspicuous committed:
Rotational Inertia Units

Trust me this is related to io layers

Something-Inconspicuous committed:
Probably decent

Something-Inconspicuous committed:
checkup

Something-Inconspicuous committed:
Feedforward added

This should be good, except for docs

Something-Inconspicuous committed:
Merge branch 'battle-o-baltimore-dev' into 75-launcher-io-layers

falOn-Dev reviewed (changes requested):
For the most part it looks good, but the flywheels need linear velocity control, and the rollers need linear position control, and position logging (it's up to you if you'd like to log in linear or angular units, but we need to be able to pass in linear)

Something-Inconspicuous committed:
Roller position, flywheel linear control

I am confused as to what is meant by "roller linear position control," as well as why you requested roller position be logged when it already is…

falOn-Dev reviewed (approved):
Looks good, merge into `battle-o-baltimore-staging` instead of `battle-o-baltimore-dev`, and start work on the subsystem
